src/api/app/models/workflow.rb

```diff
@@ -13,10 +13,10 @@ class Workflow
 
   attr_accessor :workflow_instructions, :scm_extractor_payload, :token
 
-  # Overwriting the initializer is needed to set `with_indifferent_access`
+  # Overwriting the initializer is needed to set `deep_symbolize_keys`
   def initialize(workflow_instructions:, scm_extractor_payload:, token:)
```

Comment:
We should stick to the conventions of `initialize` for an `ActiveModel::Model` (see the [code block](https://api.rubyonrails.org/classes/ActiveModel/Model.html) with `def initialize(attributes={})`). The small advantage is not having to set `@token`. It's not a blocker for this PR, but I think we should do it while we're here...
```ruby
def initialize(attributes = {})
    super
    @workflow_instructions = attributes[:workflow_instructions].deep_symbolize_keys
    @scm_extractor_payload = attributes[:scm_extractor_payload].deep_symbolize_keys
end
```

Reply:
Done. It's updated.